src/Components/Components/src/Rendering/Renderer.cs

```diff
@@ -35,19 +34,11 @@ public abstract class Renderer : IDisposable
         {
             add
             {
-                if (!(_dispatcher is RendererSynchronizationContext rendererSynchronizationContext))
```

Comment:
Renderer doesn't know anything about sync contexts now, `Dispatcher` is the all-encompassing place for threading concepts.